tests/framework/e2e/etcd_spawn.go

```diff
@@ -15,10 +15,27 @@
 package e2e
 
 import (
+	"strings"
+
 	"go.etcd.io/etcd/pkg/v3/expect"
 	"go.uber.org/zap"
 )
 
 func SpawnCmd(args []string, envVars map[string]string) (*expect.ExpectProcess, error) {
 	return SpawnCmdWithLogger(zap.NewNop(), args, envVars)
 }
+
+// SpawnCmdWithJsonPrinter spawns an etcdctl process with provided command line arguments and environment variables
```

Comment:
This function has strong assumptions about working only for `etcdctl`, this makes both name and location (`etcd_spawn.go`) not match expectations. Please consider making this a helper method for `EtcdctlV3` struct below.